.travis.yml

```diff
@@ -3,4 +3,7 @@ language: ruby
 sudo: false
 
 rvm:
-  - 2.2
+  - 2.2.2
```

Comment:
Yep, saw it too: `Gem::InstallError: rack requires Ruby version >= 2.2.2.`:grin: